content/src/fieldConfig/profile-info-poppy.json

```diff
@@ -42,6 +42,12 @@
         "header": "",
         "description": "",
         "headerNotBolded": ""
+      },
+      "nexusDbaName": {
+        "placeholder": "",
+        "header": "",
+        "errorTextRequired": "",
+        "description": ""
       }
```

Comment:
Even though it won't be used by Poppy, I believe you also need to add the nexusBusinessName block and properties here